formatters/basic/formatter_test.go

```diff
@@ -15,114 +15,256 @@
 package basic_test
 
 import (
-	"strings"
 	"testing"
 
-	"github.com/google/yamlfmt"
 	"github.com/google/yamlfmt/formatters/basic"
 	"github.com/google/yamlfmt/formatters/basic/features"
-	"github.com/google/yamlfmt/internal/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
Hi @braydonk 

I'm a random Gopher watching your repository, but I'm also one of the testify maintainers. I mention it to say I'm not against the usage of testify usually.

I'm a bit surprised you used testify and so added a bunch of dependencies to your go.mod while you have a 
github.com/google/yamlfmt/internal/assert package you could have updated slightly to provide the require.ErrorIs and require.NoError you used.